### DISCUSSION TITLE
Green Check Mark

### DISCUSSION THREAD

**Discussion by mtoretti** [davetapley/helium-tax#25] [General]
When I submit my hotspot's address, the only thing that happens is a green check pops up. Am I missing something?

**Comment by davetapley** [comment]
Hmm, it should pop up a download, like this:

https://user-images.githubusercontent.com/48232/123483431-83193500-d5bb-11eb-9308-81d06c15418f.mov

**Reply by livingsand** [reply]
@davetapley It is the same for me, instead of any calculations, a green checkmark emoji pops up and no further action is seen. I am on Windows 10, and I noticed you are on Mac OS, so that might be part of the difference. Any further help you could provide would be greatly appreciated.

**Reply by davetapley** [reply]
@livingsand sorry to hear that, thanks for reporting.
I'll double check it on Windows 10 when I get a chance.

Could you share your hotspot/wallet address with and I can see if it's that?
If you don't want to paste here publicly then shoot me an email at `dave at tapley dot com`, thanks.

**Reply by livingsand** [reply]
@davetapley Thank you for your response! I have tried my wallet, and multiple hotspots, and going off the idea that I did not start this thread, It is unlikely an error with my hotspot. Nevertheless, I am Docile Ocean Starling. As well as some more testing I did, I did at least 5 different hotspots, used Edge, Google Chrome, and Opera, and got no luck with any of them. I was really glad you are active on here because the only alternative is an expensive paid version. Again, thank you so much!

**Reply by davetapley** [reply]
@livingsand aha! Thank you so much, I think you led me to the underlying issue 🎉 

Did you have `2020` selected? If so I think this ⬇️  is the problem: 🙈 
- #28 (which I will fix ASAP)

As a workaround you _should_ be able to select `2021`, that worked for me and produced [this](https://gist.githubusercontent.com/davetapley/efa54e1778d1c66aa567bf8e4e757761/raw/43e352b6c4d232bb52ccbd29faeac6640d0ee718/11HR7T6ku14VaVU7r46NWVbC4iXMpbNPer5ubMxpQFpGtcUZWeD.csv).

**Reply by livingsand** [reply]
I forgot to mention, it also gives me a green checkmark regardless of my selection for year.... Its just.... green checkmarks..... @davetapley

**Comment by livingsand** [comment]
![image](https://user-images.githubusercontent.com/89277653/130296170-e6e248ab-0371-4e12-9c3e-aa3f0dbacb2d.png)

here is an amusing photo I got highlighting the issue @davetapley

**Reply by davetapley** [reply]
I have _no_ idea what could of cause this 😱

**Reply by livingsand** [reply]
glad you enjoyed that, every time i click "do some taxes" it creates a new green checkmark. Add a bit of time and an auto clicker.... and well....

**Reply by davetapley** [reply]
Thanks for clarifying, I think I see this one too now!

The issue is that you need to put the hotspot address in (`11HR7T6ku14VaVU7r46NWVbC4iXMpbNPer5ubMxpQFpGtcUZWeD`) and not the the name. This should definitely show ⚠️ and not ✅:
- https://github.com/davetapley/helium-tax/issues/29

**Reply by davetapley** [reply]
@livingsand okay I fixed a couple of bugs, let me know if it works (you'll have to refresh the page before trying) 🤞🏻

**Reply by livingsand** [reply]
> @livingsand okay I fixed a couple of bugs, let me know if it works (you'll have to refresh the page before trying) 🤞🏻

Tried my hotspot's name just to make sure, It gave me the proper warning. Tried my address and only took 30 seconds, showed everything properly, and then downloaded the CSV as it should. Thank you so much! I'm sure you had some fun squashing some bugs and editing code all morning and afternoon LOL. donation is on the way! I would have had to shell out 100+ dollars for a program doing what you maintain and actively fix for free, so it's the least I can do.

**Reply by davetapley** [reply]
That's great!

I'm glad it's helping others, even if it's not the most advanced too out there! 😆 

And thank you for taking the time to give feedback, hopefully this is what tripped up @mtoretti back in June 🤞🏻

**Reply by livingsand** [reply]
> That's great!
> 
> I'm glad it's helping others, even if it's not the most advanced too out there! 😆
> 
> And thank you for taking the time to give feedback, hopefully this is what tripped up @mtoretti back in June 🤞🏻

Likely, although it appears I am more motivated to pay taxes than him 😆😆
